client/src/modules/user/UserProfile/EmailNotifications.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { PureComponent } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AlertBox from 'common/components/AlertBox';
+import { saveEmailNotificationSettings } from '../model/actions';
+import { NotificationFrequency, MessageType } from 'common/constants/enums';
+import SelectDays from './SelectDays';
+import './EmailNotification.scss';
+
+// TODO: Fetch settings when profile view is displayed
+
+class EmailNotifications extends PureComponent {
+  state = {
+    success: false,
+    error: false,
+    notificationFrequency: NotificationFrequency.MONDAY,
+    areDaysVisible: true
```

Comment:
No need for that state. Please just vary on `notificationFrequency !== NotificationFrequency.NEVER`